examples/tidal_barrage/modules/input_barrages.py

```diff
@@ -0,0 +1,55 @@
+from modules.parameterisations import *
+
+
+def initialise_barrage(Barrage_Number=1):
+    """
+    :param Barrage_Number: Number of barrages considered.
```

Comment:
All methods and functions should have a doc string (in particular any user facing ones) with a one line description and a description of the arguments.